team/drp.rst

```diff
@@ -98,14 +97,31 @@ Options include:
 
 - Bouncing your connection through a `host on the Peyton network <http://www.astro.princeton.edu/docs/Hardware>`_ (this is usually the easiest way to go);
 - Making use of the `University's VPN service <https://www.net.princeton.edu/vpn/>`_.
+- Using the Research Computing gateway.
 
 If you choose the first option, you may find the ``ProxyCommand`` option to SSH helpful.
-For example, adding the follwing to :file:`~/.ssh/config` will automatically route your connection to the right place when you run :command:`ssh tiger`::
+For example, adding the following to :file:`~/.ssh/config` will automatically route your connection to the right place when you run :command:`ssh tiger`::
 
   Host tiger
-      Hostname tiger3.princeton.edu
+      Hostname tiger2-sumire.princeton.edu
       ProxyCommand ssh coma.astro.princeton.edu -W %h:%p
 
+The following SSH configuration allows access via the Research Computing gateway::
+
+    Host tigressgateway
+        HostName tigressgateway.princeton.edu
+    Host tiger* perseus* tigressdata*
+        ProxyCommand ssh -q -W %h:%p tigressgateway.princeton.edu
+
+(It may also be necessary to add a ``User`` line under ``Host tigressgateway`` if there is a mismatch between your local and Princeton usernames.)
+Entry to ``tigressgateway`` requires `2FA <https://www.princeton.edu/duoportal>`_;
+we recommend using the ``ControlMaster`` feature of SSH to persist connections, e.g.::
+
+    ControlMaster auto
+    ControlPath ~/.ssh/controlmaster-%r@%h:%p
+    ControlPersist 30d
```

Comment:
I do this (except I persist for 60s, not 30 days!), but I find it quite often causes problems — network glitches can cause the master connection to drop, but SSH doesn't always realise it and tries to reuse it for other sessions. Nothing works until I `rm ~/.ssh/master-*`.

This may be some weird peculiarity of my system (I've never tried hard to debug), but if it happens to others recommending this approach may cause more confusion than it resolves.

Reply:
Reduced to `10m`.